forte/mrdsrg-spin-integrated/master_mrdsrg.cc

```diff
@@ -177,14 +177,14 @@ void MASTER_DSRG::set_ambit_MOSpace() {
     bvirt_label_ = "V";
 
     // add Ambit index labels
-    BTF_->add_mo_space(acore_label_, "m,n,c0,c1,c2,c3,c4,c5,c6,c7,c8,c9", core_mos_, AlphaSpin);
-    BTF_->add_mo_space(bcore_label_, "M,N,C0,C1,C2,C3,C4,C5,C6,C7,C8,C9", core_mos_, BetaSpin);
-    BTF_->add_mo_space(aactv_label_, "u,v,w,x,y,z,1,2,3,a0,a1,a2,a3,a4,a5,a6,a7,a8,a9", actv_mos_,
+    BTF_->add_mo_space(acore_label_, "m,n,m1,n1,m2,n2,c0,c1,c2,c3,c4,c5,c6,c7,c8,c9", core_mos_, AlphaSpin);
```

Comment:
I prefer to keep the original indices. I think it is relatively easy to replace `m1` to `c0`, `n1` to `c1`, etc in the code.

Reply:
We can change that after the frozen orbital version is implemented, or you are simply introducing additional difficulty to my work at this stage. Imagine the equation indices are different from the one in the codes. Additional indices make the gradient code clear and easier to understand.

Reply:
Well, the equations in the paper are definitely different from the implementation now. OK, let's keep the change.